tests/test_staticfiles.py

```diff
@@ -338,6 +338,50 @@ def test_staticfiles_html_only_files(tmpdir, test_client_factory):
     assert response.text == "<h1>Hello</h1>"
 
 
+def test_staticfiles_html_200_html(tmp_path: pathlib.Path, test_client_factory):
+    path = tmp_path / "200.html"
+    path.write_text("<h1>Not found page without error</h1>")
+
+    path = tmp_path / "404.html"
+    path.write_text("<h1>Custom not found page</h1>")
```

Comment:
Why this one is created tho?

Reply:
To test that 200.html takes precedence over 404.html when both are present.